### PR TITLE
Fix a typo and try to remove VUID-VkCopyAccelerationStructureInfoKHR-dst-07792

### PR DESCRIPTION
**Pull request by FuXiii**
In `Vulkan KHR` Specification : [VUID-VkCopyAccelerationStructureInfoKHR-dst-07792](https://registry.khronos.org/vulkan/specs/1.3-khr-extensions/html/chap33.html#VUID-VkCopyAccelerationStructureInfoKHR-dst-07792)

>* VUID-VkCopyAccelerationStructureInfoKHR-dst-07792
> `dst` must be bound completely and contiguously to a single `VkDeviceMemory` object via `vkBindAccelerationStructureMemoryNV`

Is this description redundant?

When I read [NVIDIA Vulkan Ray Tracing Tutorial](https://[nvpro-samples.github.io/vk_raytracing_tutorial_KHR/](https://nvpro-samples.github.io/vk_raytracing_tutorial_KHR/)) the [source code](https://github.com/nvpro-samples/nvpro_core/blob/134eec494f6779998c0f2b2cc38088a89fd5d081/nvvk/raytraceKHR_vk.cpp#LL268C6-L268C6) .When it use `vkCmdCopyAccelerationStructureKHR` didn't called `vkBindAccelerationStructureMemoryNV` to bound a single `VkDeviceMemory` object to `VkCopyAccelerationStructureInfoKHR::dst`.

### CODE REVIEW

**Comment by FuXiii** [comment]
I try to fix the incorrect git commit `Author` name to pass the `CLAassistant` check. the check had passed. But there are many entries, it seems a bit problematic. Should I close this PR and clear my branch for concise structure？

**Comment by oddhack** [comment]
Whatever you did worked. I was thinking maybe squash and force-push to get rid of the offending commit, but not sure what would happen in that case.

**Comment by oddhack** [comment]
It is kinda weird that all those commits show up, though. No idea what's up with that.

**Comment by FuXiii** [comment]
I had checked the [Files changed](https://github.com/KhronosGroup/Vulkan-Docs/pull/2144/files), the where file changed is what I modified. And all check had passed no conflicts happen.

Please check the [VUID-VkCopyAccelerationStructureInfoKHR-dst-07792](https://registry.khronos.org/vulkan/specs/1.3-khr-extensions/html/chap33.html#VUID-VkCopyAccelerationStructureInfoKHR-dst-07792), this is where I'm confused. 

Thank you very much!

(≧∀≦)ゞ

**Comment by oddhack** [comment]
@dgkoch can you bring this up in the RT TSG? It seems to have been overlooked.

**Comment by dgkoch** [comment]
@FuXiii  can you rebase this? I'm pretty sure at least some of the drive-by fixes have been resolved.

File: chapters/accelstructures.adoc [hunk 1]
```diff
@@ -2440,14 +2439,15 @@ flink:vkCmdCopyAccelerationStructureKHR but is executed by the host.
...
+    sname:VkPhysicalDeviceAccelerationStructureFeaturesKHR::pname:accelerationStructureHostCommands>> feature must: be enabled
+
 include::{chapters}/commonvalidity/deferred_operations_common.adoc[]
   * [[VUID-vkCopyAccelerationStructureKHR-buffer-03727]]
     The pname:buffer used to create pname:pInfo->src must: be bound to
     host-visible device memory
   * [[VUID-vkCopyAccelerationStructureKHR-buffer-03728]]
     The pname:buffer used to create pname:pInfo->dst must: be bound to
-    host-visible device memory feature must: be enabled
+    host-visible device memory
```
Comment (dgkoch) [inline chapters/accelstructures.adoc:2450]: This hunk has been fixed already.

File: chapters/commonvalidity/copy_acceleration_structure_common.adoc [hunk 2]
```diff
@@ -26,8 +26,4 @@
     The range of memory backing pname:dst that is accessed by this command
     must: not overlap the memory backing pname:src that is accessed by this
     command
-  * [[VUID-{refpage}-dst-07792]]
-    pname:dst must: be bound completely and contiguously to a single
-    sname:VkDeviceMemory object via
-    flink:vkBindAccelerationStructureMemoryNV
```
[the remark this answers is not in the document]
Comment (dgkoch) [reply chapters/commonvalidity/copy_acceleration_structure_common.adoc:32]: I'll prepare a separate MR to fix this, and the remaining typo.

**Comment by oddhack** [comment]
@FuXiii @dgkoch the 1.3.282 spec update contains the internal fixes, but I'm not entirely clear if that means this PR can be closed, so please check on that.

**Comment by FuXiii** [comment]
Yes, this PR can be closed. I had checked it.

**Pull request closed without merging**